templates/app/team.phtml

```diff
@@ -10,53 +10,51 @@ $this->headTitle($title);
 
     <ul id="team">
         <li>
-            <img src="/images/team/jez.jpg" />
+            <img src="/images/team/jez.jpg"/>
             <h2>Jez Emery</h2>
             <p>
-                Jez works at Strawberrysoup working with their clients and infrastructure. Jez has been an active member of the PHP
+                Jez works at Strawberrysoup working with their clients and infrastructure. Jez has been an active member
+                of the PHP
                 Hampshire community for a few years now, and his relentless enthusiasm for building the community has
```

Comment:
Sort alignment?

Reply:
eh?

Reply:
PHP / Hampshire is split across lines unnecessarily, just a cosmetic thing!